santosfamilyfoundation/RuinaMain
# [RUIN-230] Collapsible Sections

# Description
Previously the forms listed the questions in long lists without clear section distinctions. With a collapsible section, the user will be less overwhelmed by the length of the form. It will also be easier to find a question.

# Images

## Before
![image](https://user-images.githubusercontent.com/42981026/141198659-dd2dbe94-dbd6-467d-a93b-e5132d1216e3.png)

## After
![image](https://user-images.githubusercontent.com/42981026/141197984-31a54f45-5cb2-4e05-bbe0-e284f6ce7f73.png)
![image](https://user-images.githubusercontent.com/42981026/141198017-2615100c-d66c-4e22-9ee5-9111288a54fe.png)

# Testing

1. Start a report 
2. Navigate to Crash/Road Form
3. An accordion should appear with the first section open
4. Close the section and click through the other sections to make sure all are populated with questions
